src/daemons/rhsmcertd.c

```diff
@@ -86,33 +84,21 @@ static LOG_LEVEL log_level = DEFAULT_LOG_LEVEL;
 static gboolean show_debug = FALSE;
 static gboolean run_now = FALSE;
 static gint arg_cert_interval_minutes = -1;
-static gint arg_heal_interval_minutes = -1;
 static gint arg_reg_interval_minutes = -1;
 static gboolean arg_no_splay = FALSE;
 static gboolean arg_auto_registration = FALSE;
 static int fd_lock = -1;
 
 struct CertCheckData {
     int interval_seconds;
-    bool heal;
     char *next_update_file;
 };
 
 static GOptionEntry entries[] = {
     /* marked deprecated as of 02-19-2013, needs to be removed...? */
```

Comment:
this comment belongs to the dropped `--cert-interval`, so must be removed too

Reply:
Thanks for catching this @ptoscano 
Addressed by commit: [9e27339](https://github.com/candlepin/subscription-manager/pull/3455/commits/9e2733943b276f08f981d680baf5a9c5ad6f81b7)